src/main/java/net/neoforged/neoforge/common/data/ExistingFileHelper.java

```diff
@@ -46,6 +45,7 @@
  * extra resource packs passed in via the {@code --existing} argument,
  * or mod resources via the {@code --existing-mod} argument.
  */
+@Deprecated(forRemoval = true, since = "1.21.4")
```

Comment:
Why is this deprecated for removal? There should likely be a comment explaining why, and what alternatives (if any) should be used instead.

Reply:
Existing file helper afaik is officially dead and marked for removals with no planned alternative to be implemented. There have been talks however about a follow up pr to go over to fully make EFH optional/nullable everywhere.